frontend/resources/keycloak-cbom.json

```diff
@@ -1,815 +1,1729 @@
 {
-    "bomFormat": "CycloneDX",
-    "specVersion": "1.6",
-    "serialNumber": "urn:uuid:3bf5a4a0-c24c-420c-a7c4-1af0840c8f1b",
-    "version": 1,
-    "metadata": {
-      "timestamp": "2024-07-11T13:07:30Z",
-      "tools": {
-        "services": [
-          {
-            "provider": {
-              "name": "IBM Research"
-            },
-            "name": "SonarQube Crypto Scanner Plugin"
-          }
-        ]
-      },
-      "properties": [
-        {
-          "name": "git-url",
-          "value": "https://github.com/keycloak/keycloak"
-        },
-        {
-          "name": "git-branch",
-          "value": "main"
-        },
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "serialNumber": "urn:uuid:8614bce1-98c4-492f-9fe5-eae3d7560826",
+  "version": 1,
+  "metadata": {
+    "timestamp": "2024-09-20T06:42:08Z",
+    "tools": {
+      "services": [
         {
-          "name": "commit",
-          "value": "d5041816b67d9662dfaaa284940e3965bd579dd5"
-        },
-        {
-          "name": "purl",
-          "value": "pkg:github/keycloak/keycloak"
-        },
-        {
-          "name": "purl",
-          "value": "pkg:maven/org.keycloak/keycloak-core"
```

Comment:
These additional fields (which I manually added) are used to display more information in the viewer when uploading a CBOM (which is useful for this example CBOM to appear in the interface with all its metadata)

Reply:
Ahh right, will add them